pkg/daemon/daemon.go

```diff
@@ -127,6 +128,9 @@ type Daemon struct {
 
 	// Config Drift Monitor
 	configDriftMonitor ConfigDriftMonitor
+
+	// Used for Hypershift
+	configMap string
```

Comment:
Could we use something more specific than configMap?

Reply:
I can call it hypershiftDesiredConfigMap